config/routes.rb

```diff
@@ -103,6 +103,9 @@
         end
 
         collection do
+          get 'download_error'
+          get 'download_error_path'
```

Comment:
I imagine you were trying to use a path helper to generate the path. You should stick with `download_error_file` (just "error" by itself has different connotations).

For the path helper, make sure you're in Ruby mode (`<%= ... %>`) and it'll be long, something like `download_error_file_assignment_exam_templates_path(you need to pass in the assignment id)`.